src/DotNetWorker.Core/Http/HttpRequestData.cs

```diff
@@ -62,5 +63,10 @@ public HttpRequestData(FunctionContext functionContext)
         /// </summary>
         /// <returns>The response instance.</returns>
         public abstract HttpResponseData CreateResponse();
+
+        /// <summary>
+        /// Gets the <see cref="Query"/> containing the request query. 
+        /// </summary>
+        public abstract NameValueCollection Query { get; }
```

Comment:
Would be good to have this defined as a `virtual` method in this type with the same implementation that was added to `GrpcHttpRequestData`. That would make this simpler and remove the requirement to have derived types implement this property and figure out the proper logic.

Reply:
Discussed the reasons (including breaking/impact) offline. Flagging this as resolved.